recenter_tc/plugins/modules/sector_adjusters/recenter_tc.py

```diff
@@ -270,12 +275,17 @@ def call(
     akima_only=False,
     include_archer_info=False,
 ):
+    """Recenters the TC."""
     LOG.info(
-        f"\n\n************************************************************************************"
-        f"\n************************************************************************************"
-        f"\n************************************************************************************"
-        f"\n************************************************************************************"
-        f"\n***Attempting to recenter TC sector...\n"
+        "\n\n**************************************"
+        "**********************************************"
+        "\n****************************************"
+        "********************************************"
+        "\n****************************************"
+        "********************************************"
+        "\n****************************************"
+        "********************************************"
+        "\n***Attempting to recenter TC sector...\n"
```

Comment:
@mindyls What do you think of cleaning these up some by making them:
```
log.info(
    "\n\n" + "*" * 80
    "\n" + "*" * 80
    ...
```

Or something like that? Or, alternatively, we could store the asterisks in a variable for reuse.

Reply:
That is fine with me

Reply:
Maybe open an issue, and leave this as-is for now?

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/377

Reply:
@mindyls opened nrlmmd-geoips/geoips#377 to cover this.